learning/tour-of-beam/frontend/integration_test/tour_page_test.dart

```diff
@@ -125,6 +132,30 @@ Future<void> _checkUnitContentLoadsProperly(
   );
 }
 
+Future<void> _checkSdkChangesProperly(WidgetTester wt, UnitModel node) async {
+  final defaultSdk = _getTourNotifier(wt).playgroundController.sdk;
+  final sdkCache = GetIt.instance.get<SdkCache>();
+  String? previousId;
+  for (final sdk in sdkCache.getSdks()) {
+    await _setSdk(sdk.title, wt);
+    final actualId =
+        _getTourNotifier(wt).playgroundController.selectedExample?.path;
+    expect(actualId, isNot(previousId));
```

Comment:
This is too weak. Test the SDK of the newly loaded unit or example.

Reply:
added